jersey-servers/src/main/java/com/palantir/remoting2/servers/jersey/TraceEnrichingFilter.java

```diff
@@ -91,4 +94,23 @@ public void filter(ContainerRequestContext requestContext, ContainerResponseCont
             return Optional.of(header.equals("1"));
         }
     }
+
+    private static void addUserIdMetadataToSpan(ContainerRequestContext requestContext) {
+        Optional<String> userId = tryGetUnverifiedUserId(requestContext);
+        if (userId.isPresent()) {
+            Tracer.addSpanMetadata(ImmutableMap.of(BearerTokenLoggingFilter.USER_ID_KEY, userId.get()));
+        }
+    }
+
+    private static Optional<String> tryGetUnverifiedUserId(ContainerRequestContext requestContext) {
+        String userIdKey = BearerTokenLoggingFilter.getRequestPropertyKey(BearerTokenLoggingFilter.USER_ID_KEY);
+        Object maybeUserId = requestContext.getProperty(userIdKey);
```

Comment:
not currently, @uschi2000, which is a problem. some approaches I can think of:

1. use `@Priority` annotations on the two filters. That feels ugly though since it creates an implicit relationship between these two different repos.
2. register `BearerTokenLoggingFilter` in `HttpRemotingJerseyFeature` in the order we care about. I would have to think about what happens when services are already independently registering `BearerTokenLoggingFilter`.

any thoughts?